client/my-sites/themes/sheet.jsx

```diff
@@ -141,6 +148,12 @@ const ThemeSheet = React.createClass( {
 						<div className="themes__sheet-content">
 							{ this.renderSectionNav( section ) }
 							<Card className="themes__sheet-content">{ themeContentElement }</Card>
+							<SectionHeader label={ i18n.translate( 'Features' ) } />
```

Comment:
Might be good to extract this whole part (plus the logic from above) into a `renderFeaters()` func like e.g. `renderBar()` to keep things tidy.